src/main/java/org/jitsi/jigasi/transcription/Transcriber.java

```diff
@@ -336,6 +371,30 @@ public AudioMixerMediaDevice getMediaDevice()
     }
 
     /**
+     * If all participant have been completely transcribed, and state is
+     * FINISHING_UP, the state can be set to FINISHED
+     */
+    private void isFinishedUp()
```

Comment:
This is confusing -- the name starts with "is", but it doesn't return a boolean? Can you please rename and clarify the doc?